pages/opam-using.html

```diff
@@ -191,4 +177,10 @@ <h3>Roots</h3>
 eval $(opam env)
 </code></pre>
 
+
+<h2>I still need help!</h2>
+
+If the above does not work for you, or you are confused, then feel free to open a topic on <a href="https://coq.discourse.group">Discourse</a> or <a href="/community">our other communication channels</a>.
```

Comment:
Note that for troubleshooting I would be inclined to recommend Zulip over Discourse as it is better suited for the kind of high traffic interactions that usually comes with debugging a particular setup.

```suggestion
<p>If the above does not work for you, or you are confused, then feel free to open a topic on <a href="https://coq.discourse.group">Discourse</a> or <a href="/community">our other communication channels</a>.</p>
```